open_spiel/games/trade_comm.h

```diff
@@ -96,7 +96,7 @@ class TradeCommGame : public Game {
     return std::unique_ptr<State>(
         new TradeCommState(shared_from_this(), num_items_));
   }
-  int MaxChanceOutcomes() const override { return num_items_; }
+  int MaxChanceOutcomes() const override { return num_items_ * num_items_; }
```

Comment:
This change is based on:

https://github.com/deepmind/open_spiel/blob/master/open_spiel/games/trade_comm.cc#L234

Reply:
Yes this is right, nice catch!